docarray/index/backends/weaviate.py

```diff
@@ -715,8 +719,26 @@ def python_type_to_db_type(self, python_type: Type) -> Any:
             bool: 'boolean',
             np.ndarray: 'number[]',
             bytes: 'blob',
+            list: 'text[]',
         }
 
+        if get_origin(python_type) == list:
+            py_weaviate_list_type_map = {
+                int: 'number[]',
+                float: 'number[]',
```

Comment:
a list of int in python should be mapped to 'int[]' in weaviate, since 'number[]' is for float64